AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/RemoteSyncEngine.swift

```diff
@@ -270,6 +272,9 @@ class RemoteSyncEngine: RemoteSyncEngineBehavior {
         // Hold a reference so we can `reset` while initial sync is in process
         self.initialSyncOrchestrator = initialSyncOrchestrator
 
+        syncEventEmitter = SyncEventEmitter(initialSyncOrchestrator: initialSyncOrchestrator,
+                                            reconciliationQueue: reconciliationQueue ?? nil)
```

Comment:
i think you can pass the optional in as is `reconciliationQueue: reconciliationQueue)` without nil coalescing

Reply:
removed nil